handler/src/main/java/io/netty/handler/ssl/OpenSslClientContext.java

```diff
@@ -176,21 +299,32 @@ public OpenSslClientContext(File trustCertCollectionFile, TrustManagerFactory tr
         this(toX509CertificatesInternal(trustCertCollectionFile), trustManagerFactory,
                 toX509CertificatesInternal(keyCertChainFile), toPrivateKeyInternal(keyFile, keyPassword),
                 keyPassword, keyManagerFactory, ciphers, cipherFilter, apn, null, sessionCacheSize,
-                sessionTimeout, false);
+                sessionTimeout, false, KeyStore.getDefaultType());
+    }
+
+    OpenSslClientContext(X509Certificate[] trustCertCollection, TrustManagerFactory trustManagerFactory,
```

Comment:
Since this is package-private, modify the existing this existing constructor instead of creating a new one.